Orange/widgets/classify/owclassificationtree.py

```diff
@@ -65,6 +65,26 @@ def get_learner_parameters(self):
             ("maximum depth {}".format(self.max_depth), self.limit_depth)) if c) or "None"
         return items
 
+    def init_code_gen(self):
+        def run():
+            ow.set_data(input_data)
```

Comment:
If possible, I would suggest submitting repr cahnges as a separate PR (create a new branch from the latest master, cherry-pick the relevant commits and submit a PR). That change could be merged before you are done with the code generator.